definitions/variable/industry/tag_nonfer_commodities.yaml

```diff
@@ -0,0 +1,13 @@
+- Non-Ferrous Metals Commodity:
+    - Aluminum:
```

Comment:
As another subcategory of aluminum, we could add bauxite here similar to including iron ore.